ci/jenkins/jobs/macros.yaml

```diff
@@ -275,3 +275,55 @@
           set -e 
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-mc.sh --testcase e2e --registry ${DOCKER_REGISTRY} --mc-gateway --codecov-token "${CODECOV_TOKEN}" --coverage --kind
+- builder:
+    name: builder-ipv6-e2e-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/jenkins/test.sh --testcase e2e --registry ${DOCKER_REGISTRY} --testbed-type jumper
+
+- builder:
+    name: builder-ipv6-conformance-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/jenkins/test.sh --testcase conformance --registry ${DOCKER_REGISTRY} --testbed-type jumper
+- builder:
+    name: builder-ipv6-networkpolicy-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/jenkins/test.sh --testcase networkpolicy --registry ${DOCKER_REGISTRY} --testbed-type jumper
+- builder:
+    name: builder-ds-networkpolicy-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          sed -i "s/\$(DOCKER_BUILD_ARGS) ./\$(DOCKER_BUILD_ARGS) . --network host/g" Makefile
+          ./ci/jenkins/test.sh --testcase networkpolicy --registry ${DOCKER_REGISTRY} --testbed-type jumper
+- builder:
+    name: builder-ds-e2e-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          sed -i "s/\$(DOCKER_BUILD_ARGS) ./\$(DOCKER_BUILD_ARGS) . --network host/g" Makefile
+          ./ci/jenkins/test.sh --testcase e2e --registry ${DOCKER_REGISTRY} --testbed-type jumper
+- builder:
+    name: builder-ds-conformance-jumper
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -e
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          sed -i "s/\$(DOCKER_BUILD_ARGS) ./\$(DOCKER_BUILD_ARGS) . --network host/g" Makefile
+          ./ci/jenkins/test.sh --testcase conformance --registry ${DOCKER_REGISTRY} --testbed-type jumper
```

Comment:
Missing new line at the end.

Reply:
Added a new line to the end of the file.